ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml

```
@@ -65,6 +65,17 @@ images:
     openshift_release_golang-1.17:
       as:
       - registry.ci.openshift.org/openshift/release:golang-1.17
+  to: assisted-service-build-base
+- dockerfile_literal:
+    FROM base-build
+    COPY . .
```

```suggestion
    COPY . .
```
I think it can be removed since the sources are already copied below

---

https://github.com/openshift/ci-tools/blob/6e7726525c19c29eebe8e3f970d0c1c01e2f4d02/pkg/api/types.go#L1822-L1824 

Emphasis on "context"

---

But on the other hand... https://github.com/openshift/ci-tools/blob/6e7726525c19c29eebe8e3f970d0c1c01e2f4d02/pkg/api/types.go#L1838-L1839 "directory in the destination image"

So :shrug: